### ISSUE TITLE
TypeError: __init__() missing 1 required positional argument: 'descriptor_pool'

### ISSUE THREAD

**Issue by lftalkington** [Clarifai/clarifai-python#101]
Trying to use your product and running into this issue: OSX Mojave 

clarifai==2.6.0

Any ideas?

``` 
from clarifai.rest import ClarifaiApp 
app = ClarifaiApp(api_key='mykeyhere')                                                            
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-9f98bbfcf347> in <module>
----> 1 app = ClarifaiApp(api_key='<REDACTED>')

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in __init__(self, app_id, app_secret, base_url, api_key, quiet, log_level)
    122     self.concepts = Concepts(self.api)  # type: Concepts
    123     self.inputs = Inputs(self.api)  # type: Inputs
--> 124     self.models = Models(self.api, self.solutions)  # type: Models
    125     self.workflows = Workflows(self.api)  # type: Workflows
    126 

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in __init__(self, api, solutions)
   1061     # the cache of the model name -> model id mapping
   1062     # to avoid an extra model query on every prediction by model name
-> 1063     self.model_id_cache = self.init_model_cache()
   1064 
   1065   def init_model_cache(self):

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in init_model_cache(self)
   1079 
   1080     try:
-> 1081       for m in models:
   1082         model_name = m.model_name
   1083         model_type = m.output_info['type']

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in get_all(self, public_only, private_only)
   1185 
   1186     while True:
-> 1187       res = self.api.get_models(page, per_page)
   1188 
   1189       if not res['models']:

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in get_models(self, page, per_page)
   3936 
   3937     response = self._grpc_request(self._grpc_stub().ListModels,
-> 3938                                   ListModelsRequest(page=page, per_page=per_page))
   3939     return response
   3940 

/anaconda3/lib/python3.6/site-packages/clarifai/rest/client.py in _grpc_request(self, method, argument)
   3640     for attempt_num in range(1, max_attempts + 1):
   3641       try:
-> 3642         res = method(argument)
   3643 
   3644         dict_res = protobuf_to_dict(res)

/anaconda3/lib/python3.6/site-packages/clarifai/rest/grpc/grpc_json_channel.py in __call__(self, request, metadata)
    169     # Get the actual message object to construct
    170     message = self.response_deserializer
--> 171     result = dict_to_protobuf(message, response_json, ignore_unknown_fields=True)
    172 
    173     return result

/anaconda3/lib/python3.6/site-packages/clarifai/rest/grpc/custom_converters/custom_dict_to_message.py in dict_to_protobuf(protobuf_class, js_dict, ignore_unknown_fields)
      8   # type: (type(Message), dict, bool) -> Message
      9   message = protobuf_class()
---> 10   parser = _CustomParser(ignore_unknown_fields=ignore_unknown_fields)
     11   parser.ConvertMessage(js_dict, message)
     12   return message

TypeError: __init__() missing 1 required positional argument: 'descriptor_pool'
```

**Comment by rok-povsic** [comment]
@lftalkington Hi, a Protobuf version 3.7.0 was released just today which is the reason for this error. We're looking into how to fix this. At this moment, you can manually install the previous version with `pip install protobuf==3.6.1`.

**Comment by ChrisBolman** [comment]
thanks @rok-povsic, one of our engineers just ran into this too. appreciate the suggestion.

**Comment by kevindalias** [comment]
@ChrisBolman I just encountered the same issue. Downgrading to protobuf==3.6.1 didn't do it for me due to another library's dependency, but I was able to get `ClarifaiApp()` working again by overriding the `__init__()` method in `_CustomParser` to match the older version of protobuf. 

```
class _CustomParser(_Parser):

  def __init__(self, ignore_unknown_fields):
    """
    Overriding the super class init. More details here:
    https://github.com/Clarifai/clarifai-python/issues/101
    """
    self.ignore_unknown_fields = ignore_unknown_fields

  def _ConvertFieldValuePair(self, js, message):
    """
    Because of fields with custom extensions such as cl_default_float, we need
    to adjust the original's method's JSON object parameter by setting them explicitly to the
    default value.
    """

    message_descriptor = message.DESCRIPTOR
    for f in message_descriptor.fields:
      default_float = f.GetOptions().Extensions[extensions_pb2.cl_default_float]
      if default_float:
        if f.name not in js:
          js[f.name] = default_float

    super(_CustomParser, self)._ConvertFieldValuePair(js, message)
```

`_CustomParser` is here: `clarifai/rest/grpc/custom_converters/custom_dict_to_message.py`

Hope this helps others.

**Comment by rok-povsic** [comment]
@lftalkington @ChrisBolman @kevindalias We've just released the Protobuf 3.7.0 support. Please upgrade to the latest Clarifai version with: `pip install clarifai --upgrade`.

Thanks a lot for reporting this.
